mirrord/kube/src/api/container/job.rs

```diff
@@ -129,29 +132,44 @@ where
         self.inner.params()
     }
 
-    fn as_update(&self) -> Result<Self::Update> {
+    fn as_update(&self) -> Self::Update {
```

Comment:
This was already the case, but, why is this function called `as_update`?

Reply:
No idea :eyes: 
Possible interpretation - an update to cluster state?